UnitsNet.Tests/CustomCode/RotationalStiffnessTests.cs

```diff
@@ -28,6 +28,8 @@ namespace UnitsNet.Tests.CustomCode
     public class RotationalStiffnessTests : RotationalStiffnessTestsBase
     {
         protected override double NewtonMetersPerRadianInOneNewtonMeterPerRadian => 1;
+        protected override double PoundForceFeetPerDegreesInOneNewtonMeterPerRadian => 0.0128729;
```

Comment:
I couldn't find a second source to verify, but your google result looks good to me.